src/MyEventEmitter.js

```diff
@@ -1,12 +1,119 @@
 'use strict';
 
-export class MyEventEmitter {
-  on() {}
-  once() {}
-  off() {}
-  emit() {}
-  prependListener() {}
-  prependOnceListener() {}
-  removeAllListeners() {}
-  listenerCount() {}
+class MyEventEmitter {
+  constructor() {
+    this.listeners = {};
+  }
+
+  on(eventName, callBack) {
+    if (typeof callback !== 'function' || typeof eventName !== 'string') {
+      throw new Error();
+    }
+
+    if (this.listeners.hasOwnProperty(eventName)) {
+      this.listeners[eventName].push(callBack);
+    } else {
+      this.listeners[eventName] = [ callBack ];
+    }
```

Comment:
Make sense to use ternary operator here as it will make a code shorter 
Check other places